src/Parallel/centrality/stress.jl

```diff
@@ -16,28 +22,45 @@ function distr_stress_centrality(g::AbstractGraph,
     stress = @distributed (+) for s in vs
         temp_stress = zeros(Int64, n_v)
         if degree(g, s) > 0  # this might be 1?
-            state = LightGraphs.dijkstra_shortest_paths(g, s; allpaths=true, trackvertices=true)
+            state = LightGraphs.dijkstra_shortest_paths(
+                g,
+                s;
+                allpaths = true,
+                trackvertices = true,
+            )
```

Comment:
This goes overboard, I think. The line is short enough not to spread it over multiple lines. Can we have a rule that says: if the line length < 100, we allow single-line?